combined_browser_metrics/explores/active_users_aggregates.explore.lkml

```diff
@@ -3,7 +3,8 @@ include: "/shared/views/*"
 
 explore: active_users_aggregates {
   always_filter: {
-    filters: [active_users_aggregates.submission_date: "this year"]
+    filters: [active_users_aggregates.app_name: "Firefox Desktop, Fenix, Fenix BrowserStack, Firefox iOS, Firefox iOS BrowserStack, Focus Android,  Focus iOS",
+      active_users_aggregates.submission_date: "after 4 weeks ago"]
```

Comment:
This is great, only query the full year where needed.

Reply:
Yeah... I think the person can decide how far back they want to go. With the current `this year` I think a lot of people wouldn't bother changing it and run a query while they are still trying to understand the data. So I was thinking it's better to make that window smaller to begin with.